### PR TITLE
refactor: Replaces deprecated errors module with fmt

### PR DESCRIPTION
**Pull request by MoritzThomasHuebner**
refactoring: Replaced calls to `errors.Wrap` and  `errors.Wrapf` with `fmt.Errorf`

## Summary

This PR starts the process of removing the uses of [github.com/pkg/errors](github.com/pkg/errors) which [is no longer maintained](https://github.com/pkg/errors/issues/245). However, this change will remove stack traces in the output. 

The issue was raised in #12632. The remaining uses of "github.com/pkg/errors" are removed in #12683. Splitting these PRs should make them easier to review. 

This PR:
1. Replaces calls of the form `errors.Wrap(err, wrap)` with `fmt.Errorf(wrap + ": %w", err)` (or similar)
2. Replaces calls of the form `errors.Wrapf(err, wrap, ...vars)` with `fmt.Errorf(wrap + ": %w", ...vars, err)`

Go also automatically formatted some files while I was making these changes. There are a few white spaces that were added or removed unrelated to the change in error wrapping due to automatic formatting.

I attached an example below of the change in output. I obtained this by modifying the source code so a test would fail.

Old output:
```
    eap_aka_test.go:48: 
        	Error Trace:	eap_aka_test.go:48
        	Error:      	Received unexpected error:
        	            	I caused this error deliberately.
        	            	Error validating EAP packet
        	            	magma/cwf/gateway/services/uesim/servicers.(*UESimServer).HandleEap
        	            		/home/bmoritz/Documents/magma/cwf/gateway/services/uesim/servicers/eap.go:36
        	            	magma/cwf/gateway/services/uesim/servicers_test.TestEapAkaIdentityRequest
        	            		/home/bmoritz/Documents/magma/cwf/gateway/services/uesim/servicers/eap_aka_test.go:47
        	            	testing.tRunner
        	            		/usr/local/go/src/testing/testing.go:1439
        	            	runtime.goexit
        	            		/usr/local/go/src/runtime/asm_amd64.s:1571
```

New output
```
    eap_aka_test.go:48: 
        	Error Trace:	eap_aka_test.go:48
        	Error:      	Received unexpected error:
        	            	Error validating EAP packet: I caused this error deliberately.
        	Test:       	TestEapAkaIdentityRequest
```

## Test Plan

Run unit tests.

## Additional Information

This is my first PR!

- [ ] This change is backwards-breaking

### CODE REVIEW

File: src/go/agwd/server/server.go [hunk 1]
```diff
@@ -76,18 +75,15 @@ func cleanupUnixSocket(
...
 	return nil
 }
 
 func cleanupUnixSocketOrDie(logger log.Logger, path string) {
 	if err := cleanupUnixSocket(logger, os.Stat, os.Remove, net.DialTimeout, path); err != nil {
-		panic(errors.Wrapf(
-			err,
-			"cleanupUnixSocket(logger=_, target.Endpoint=%s)",
-			path))
+		panic(fmt.Errorf("cleanupUnixSocket(logger=_, target.Endpoint=%s): %w", path, err))
```
Comment (Neudrino) [inline src/go/agwd/server/server.go]: While in general cleanup is nice, we might not want to change this in the same commit and PR, as the PR is already big enough. (Also lines below.)

File: cwf/gateway/integ_tests/test_runner.go [hunk 2]
```diff
@@ -477,32 +479,32 @@ func (tr *TestRunner) WaitForEnforcementStatsForRuleGreaterThanOrDoesNotExist(im
...
+		return nil, true
 	}
+	txBytes := record.BytesTx
+	if record.BytesTx < min {
+		return record, false
+	}
+	fmt.Printf("\t\u2713 %s, %s now passed %d > %d in enforcement stats!(%d%%)\n",
+		imsi, ruleID, txBytes, min, 100*txBytes/min)
+	return record, true
+}
```
Comment (Neudrino) [inline cwf/gateway/integ_tests/test_runner.go]: This seems to be a white-space change only. While in general cleanup is nice, we might not want to change this in the same commit and PR, as the PR is already big enough. (Also lines below.)

File: cwf/gateway/integ_tests/test_runner.go [hunk 3]
```diff
@@ -306,19 +306,21 @@ func (tr *TestRunner) GenULTrafficBasedOnPolicyUsage(req *cwfprotos.GenTrafficRe
...
+		if newVolume < 1000 {
+			newVolume = 1000
+		}
 		newVolumeStr := fmt.Sprintf("%dK", newVolume/1000)
 
 		req.Volume = &wrappers.StringValue{Value: newVolumeStr}
 		fmt.Printf("- not enough traffic genereted, sending %dKB more. Will be around %d%% of volume requested\n",
 			newVolume/1000,
-			100 * (record.BytesTx+newVolume)/totalVolume,
+			100*(record.BytesTx+newVolume)/totalVolume,
```
Comment (Neudrino) [inline cwf/gateway/integ_tests/test_runner.go]: Those changes of white-spaces around operators seem rather inconsistent and every PR seems to want to change them. I think in this PR, we should not touch them.

File: src/go/agwd/server/server.go (hunk 1 above)
Comment (MoritzThomasHuebner) [reply src/go/agwd/server/server.go]: This line and other instances of this in the file also change `errors.Wrapf` to `fmt.Errorf`. The reformatting was incidental.

File: cwf/gateway/integ_tests/test_runner.go (hunk 2 above)
Comment (MoritzThomasHuebner) [reply cwf/gateway/integ_tests/test_runner.go]: I removed this white space change.

File: cwf/gateway/integ_tests/test_runner.go (hunk 3 above)
Comment (MoritzThomasHuebner) [reply cwf/gateway/integ_tests/test_runner.go]: I removed this white space change.

**Pull request closed without merging**